tests/metrics/test_nlp_metrics.py

```diff
@@ -200,3 +201,22 @@ def test_perplexity():
     perplexity = perplexity_metric.compute()
 
     assert torch.equal(torch.exp(ce), perplexity)
+
+
+def test_in_context_learning_lm_accuracy(tiny_gpt2_tokenizer):
+    pad = tiny_gpt2_tokenizer.eos_token_id
```

Comment:
The test fixture actually has a pad token, so I think you should be able to use that: https://github.com/mosaicml/composer/blob/bb2b12bd0235318325186a542904669fd4616863/tests/fixtures/fixtures.py#L170